media-libs/vulkan-loader/vulkan-loader-9999.ebuild

```diff
@@ -32,7 +33,7 @@ multilib_src_configure() {
 		-DCMAKE_SKIP_RPATH=True
 		-DBUILD_TESTS=False
```

Comment:
what about when you actually want to run tests?

Reply:
i didnt write that part, but i added the use